tests/integration/test_users.py

```diff
@@ -1,151 +0,0 @@
-from models.user import UserModel
-from conftest import is_valid, log
-from freezegun import freeze_time
-from models.user import RoleEnum
-
-def test_user_auth(client, test_database, admin_user):
```

Comment:
I'm not sure why this file is being removed.  if it is not needed perhaps we can do this in a separate PR.